presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -34,6 +35,7 @@
     private boolean storageBasedBroadcastJoinEnabled;
     private DataSize storageBasedBroadcastJoinWriteBufferSize = new DataSize(24, MEGABYTE);
     private String storageBasedBroadcastJoinStorage = "local";
+    private DataSize posBroadcastJoinMaxMemory = new DataSize(Runtime.getRuntime().maxMemory() * 0.1, BYTE);
```

Comment:
Ditto about naming: `sparkBroadcastJoinMaxMemoryOverride`. Also let's keep it `null` by default (not set)